docs/docs/behaviors/mod-morph.md

```diff
@@ -71,3 +65,30 @@ Example:
 ```
 mods = <(MOD_LGUI|MOD_LSFT|MOD_RGUI|MOD_RSFT)>;
 ```
+
+### Advanced configuration
+
+`masked_mods`
```

Comment:
I believe we prefer hyphenated property names by convention, so `masked-mods` would be the right format for the docs (and yaml/dtsi).

Reply:
done